bundles/pom.xml

```diff
@@ -5,684 +5,1349 @@
   <modelVersion>4.0.0</modelVersion>
```

Comment:
File to be fixed.

Reply:
Rebased

Reply:
POM file is still not good with a lot of added blank lines.

Reply:
You removed the final return line.

Reply:
Now it is ok